docs/products/postgresql/howto/use-pg-audit-logging.md

```diff
@@ -271,12 +273,20 @@ curl --request POST \
 
 ## Visualize your logs
 
-If your logs are available in Aiven for OpenSearch, use
-[OpenSearch Dashboards](/docs/products/opensearch/dashboards/get-started) to visualize the
-logs.
+Choose one of the
+[tools or methods for visualizing your collected audit logs](/docs/products/postgresql/concepts/pg-audit-logging#collecting-and-visualizing-logs).
+
+Example: **[OpenSearch Dashboards](/docs/products/opensearch/dashboards/get-started)**
 
-To preview your audit logs in OpenSearch Dashboards, use the filtering tool: select
-`AIVEN_AUDIT_FROM`, set its value to `pg`, and apply the filter.
+1. Integrate your Aiven for PostgreSQL with Aiven for OpenSearch.
```

Comment:
I think there aren't generic integration steps in the docs, so maybe it'd be worth adding the steps for integrating here? Or linking to the create integrations example?